  Generating xr_generated_dispatch_table.c using C:/Program Files (x86)/Microsoft Visual Studio/Shared/Python39_64/python.exe on utility_source_generator.py
  C:/Program Files (x86)/Microsoft Visual Studio/Shared/Python39_64/python.exe: can't open file 'C:\Users\Chengxiang Li\Desktop\OpenXR-SDK-Source-release-1.1.37\src\scripts\src_genxr.py': [Errno 2] No such file or directory
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: Custom build for 'D:\HelloWorldTest\CMakeFiles\fcf03554df7e2a0ceeb9c61033fc6f0f\xr_generated_dispatch_table.c.rule;D:\HelloWorldTest\CMakeFiles\fcf03554df7e2a0ceeb9c61033fc6f0f\xr_generated_dispatch_table.h.rule;D:\HelloWorldTest\CMakeFiles\2f7c988a15c3f6804767b0ee0132f01c\xr_common_generated_files.rule;C:\Users\Chengxiang Li\Desktop\OpenXR-SDK-Source-release-1.1.37\src\CMakeLists.txt' exited with code 2.
